places/london-uk.md

```diff
@@ -0,0 +1,10 @@
+## London, UK
+
+### Places to go
+- Savoy Grill
+- Meat Licqour
+- Dinner (Heston)
```

Comment:
Well, to make it easier. Do you know how long it takes to write Pietermaritzburg?!